packages/reports/src/sass/app/_common/components/allow-equals.scss

```diff
@@ -0,0 +1,19 @@
+/** @define allow-equals */
```

Comment:
Some of the scss in this folder may not be required. Please check.

Reply:
will check :pray: